features/step_definitions/setup_steps.rb

```diff
@@ -280,7 +280,7 @@ def delete_all_draft_briefs (user_id)
     brief_id = brief["id"]
     updated_by = {updated_by: "Functional tests"}
 
-    if brief["status"] != "live"
+    if !["live", "closed"].include? brief["status"]
```

Comment:
I think this would be more idiomatic as `unless ["live", "closed"].include? brief["status"]`